tools/@aws-cdk/node-bundle/package.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "@aws-cdk/node-bundle",
```

Comment:
I thought we were going to call this `node-bundle-cli`.

Also, does it need to be in the `@aws-cdk` namespace? I would argue no.

Reply:
> I thought we were going to call this node-bundle-cli.

Yeah, when we last spoke about naming, the tool was assuming it can simply take the entry point mentioned in the `bin` directive of the `package.json`, which made it somewhat CLI specific. The problem is that `bin` entries don't necessarily point to `.js` file (for [example](https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/package.json#L8) in the CDK itself). It's actually pretty tricky to deduce which `.js` files need to bundled, so I eventually opted for this to be a required config. Doing so meant removing assumption about `bin` directive, making it not CLI specific. BTW requiring an entry-point be passed as a config also removes our need to auto-generate entrypoints and start linking to them. It now simply replaces the given entry-point with its bundled version, and it just works. 

> Also, does it need to be in the @aws-cdk namespace? I would argue no.

When we publish this, definitely not. But for now, while its in the CDK repo, I remember us explicitly moving all tools to be in the `@aws-cdk/` scope because of some AppSec concerns (@RomainMuller I believe you handled this right? can you share some details?)

Reply:
Here is reasoning for the scope changes: https://github.com/aws/aws-cdk/pull/16689

Reply:
The `@aws-cdk/` namespace is only required for `private` packages - since we don't publish these, but they are still appearing in our package's `devDependencies` declarations, there _is_ a supply chain hijacking vector here (although only on very specific/unusual workflows).

That package here is NOT marked `private` (although it should probably be - as that hasn't gone through trademark review yet), it will be published and hence cannot be seized by a third-party... we'd be at liberty to name this however we please.

Reply:
Good catch, I will make it private